### PR TITLE
Updated neutralizing background and truncated Ewald 

### PR DESCRIPTION
**Pull request by bjornstenqvist**

### CODE REVIEW

File: coulombgalore.h [hunk 1]
```diff
@@ -1183,10 +1182,10 @@ template <class T, bool debyehuckel = true> class EnergyImplementation : public
      * @warning Not tested!
```
Comment (mlund) [inline coulombgalore.h]: Should this warning still be here?

File: coulombgalore.h [hunk 2]
```diff
@@ -1253,12 +1252,17 @@ class Ewald : public EnergyImplementation<Ewald> {
             eps_sur = infinity;
         double Q = 1.0 - std::erfc(eta) - 2.0 * eta / pi_sqrt * std::exp(-eta2); // Eq. 12 in DOI: 10.1016/0009-2614(83)80585-5 using 'K = cutoff region'
         T0 = (std::isinf(eps_sur)) ? Q : ( Q - 1.0 + 2.0 * (eps_sur - 1.0) / (2.0 * eps_sur + 1.0) ); // Eq. 17 in DOI: 10.1016/0009-2614(83)80585-5
-        chi = 4.0 * ( 0.5 * ( 1.0 - zeta ) * std::erfc( eta + zeta / ( 2.0 * eta ) ) * std::exp( zeta ) + std::erf( eta ) * std::exp(-zeta2 / ( 4.0 * eta2 ) ) + 
-                0.5 * ( 1.0 + zeta ) * std::erfc( eta - zeta / ( 2.0 * eta ) ) * std::exp( -zeta ) - 1.0 ) * pi * cutoff2 / zeta2;
-        // chi = -pi * cutoff2 / eta2 according to DOI:10.1021/ct400626b, for uncscreened system
         zeta = cutoff / debye_length;
         zeta2 = zeta * zeta;
         zeta3 = zeta2 * zeta;
+        if(zeta < 1e-6) {
```
Comment (mlund) [inline coulombgalore.h]: add code comments to explain the logic

**Review by mlund — APPROVED** [review]

File: coulombgalore.h (hunk 1 above)
Comment (bjornstenqvist) [reply coulombgalore.h]: No indeed. Has now been changed.

File: coulombgalore.h (hunk 2 above)
Comment (bjornstenqvist) [reply coulombgalore.h]: Has made a comment about it. OK?

**Pull request merged**